### PR TITLE
[CID-877] - Adding documentation for password policies

### PR DESCRIPTION
**Pull request by rackerbob**
**Quality audit reminder**

Doc team: While logged in using your O365 account, complete the quality checklist at http://bit.ly/2hwvBKX. Use the checklist for significant doc changes or additions submitted by you or a non-doc staff contributor.

### CODE REVIEW

File: api-docs/api-reference/methods/domains/get-password-policy-on-domain-v2.0.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    X-Auth-Token, String *(Required)*, A valid authentication token.
+
+This table shows the URI parameters for the request:
+
+.. csv-table::
+    :header: Name, Type, Description
+    :widths: 2, 2, 2
+
+    ``{domainId}``, String *(Required)*, A domain ID.
```
Comment (timcline) [inline api-docs/api-reference/methods/domains/get-password-policy-on-domain-v2.0.rst]: Based on what Jorge told me yesterday, we are not using backticks to URI parameters. This is the page he referred me to as documentation for the conventions https://github.rackspace.com/ServiceAPIContracts/global-auth-keystone-extensions/blob/master/docs/samples/method-example.rst

File: api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    X-Auth-Token, String *(Required)*, A valid authentication token.
+
+This table shows the URI parameters for the request:
+
+.. csv-table::
+    :header: Name, Type, Description
+    :widths: 2, 2, 2
+
+    ``{domainId}``, String *(Required)*, A domain ID.
```
Comment (timcline) [inline api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst]: Same as comment above. I think we want to remove the backticks

File: api-docs/api-reference/methods/domains/get-password-policy-on-domain-v2.0.rst (hunk 1 above)
Comment (rackerbob) [inline api-docs/api-reference/methods/domains/get-password-policy-on-domain-v2.0.rst]: fixed

File: api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst (hunk 2 above)
Comment (rackerbob) [inline api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst]: fixed

File: api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+.. note::
+
+    If the password on a user's account has been updated since
+    June 8, 2017, Identity uses the date and time
+    of the password change to determine whether the password has expired;
+    otherwise, Identity uses the last time any attribute on the account
+    was updated (including password, email, and MFA settings).
+
+    Password policies can only be set using JSON; XML is not supported
```
Comment (cyrichardson) [inline api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst]: Add a period at the end of this sentence.

Comment (rackerbob) [inline api-docs/api-reference/methods/domains/set-password-policy-on-domain-v2.0.rst]: fixed

**Pull request merged**